rudder-docker.yml

```diff
@@ -27,6 +27,9 @@ services:
       - CONFIG_BACKEND_URL=https://api.rudderlabs.com
       - WORKSPACE_TOKEN=<your_workspace_token>
       - CONFIG_PATH=/app/config/config.toml
+    # Uncomment the following lines to mount workspaceConfig file
+    # volumes:
```

Comment:
We also need to update the env variable - RSERVER_BACKEND_CONFIG_CONFIG_FROM_FILE in docker.env
Let's also mention it here in the comment. So that the user don't miss it.

Reply:
We should also mention in the docker.env to update the docker-compose and then mount the file at that location.

Reply:
RSERVER_BACKEND_CONFIG_CONFIG_FROM_FILE -> DONE